### ISSUE TITLE
Document tooling landscape

### ISSUE THREAD

**Issue by davidyezsetz** [interactive-pioneers/conventions#70]
It might be a good idea to give a brief intro to the current tooling landscape?

**Comment by ain** [comment]
I agree. We should deliver experience in documentation. What in our experience is actually working out in conjunction of all the conventions outlined.

**Comment by davleh** [comment]
I think this is a good idea. I started putting links to repositories for linter plugins (atom and sublime) for example.
